config/cable.yml

```diff
@@ -6,5 +6,7 @@ test:
 
 production:
   adapter: redis
-  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  url: <%= Rails.application.secrets.redis_url %>
   channel_prefix: wheel_production
+  ssl_params:
+    verify_mode: <%= OpenSSL::SSL::VERIFY_NONE %>
```

Comment:
In reference to https://github.com/bigbinary/wheel/pull/609#discussion_r716548331.

My previous comment was based on this. This was already added in the PR. So I thought you were trying to avoid redundancy by keeping it in secrets.yml. And I thought you had missed out using these secrets in `sidekiq.rb`.

Given that you have mentioned that we don't need to add to `secrets.yml`, I think we can remove these lines.

Reply:
@sandip-mane My bad. Just now saw the file name is `cable.yml`. Ignore my comment then.